src/components/select/select.bemhtml.js

```diff
@@ -6,7 +6,7 @@ block('select')(
         return applyNext();
 
         const React = require('react');
-        const Popup = require('../src/components/popup/popup');
+        const Popup = require('../src/components/popup/popup.jsx');
```

Comment:
because we use RenderInBody component in Popup render method. Of course, we can rewrite it to the normal js code. But should we do it?